lib/scrape.js

```diff
@@ -102,6 +102,6 @@ async function fetchOrgsWithData() {
   data.sort(
     (a, b) => b.completed_task_instance_count - a.completed_task_instance_count
   )
-
-  fs.writeFileSync('out/data.json', JSON.stringify(data))
+  
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.whitespace`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpz_acekp1/lib/scrape.js
+++ b/tmp/tmpz_acekp1/lib/scrape.js
@@ -102,6 +102,6 @@
   data.sort(
     (a, b) => b.completed_task_instance_count - a.completed_task_instance_count
   )
-  
+
   fs.writeFileSync(__dirname + '/../out/data.json', JSON.stringify(data))
 })()
```